bin/deploy-vhost

```diff
@@ -236,7 +238,11 @@ sub nginx_graceful {
 
 sub graceful {
     # Prompt webserver to reread files, on machines which have apache...
-    apache_graceful();
+    if (!$conf->{touch_to_restart} || $restart_apache) {
```

Comment:
Yes, that is so that it maintains the current behaviour with any vhost that doesn't use the new `touch_to_restart` (and so is assuming that Apache gets restarted, which will be necessary to reload new code, otherwise it'll just stick in memory).